src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -35,6 +35,7 @@ public enum Mode
             IReadOnlyList<string> trustedHttpsCertificates,
             string apiKey,
             string searchBaseUrl,
+            string autoCompleteUrl,
```

Comment:
This can be implied by `searchBaseUrl` right? I don't see the value in the additional configuration.